olumidae/WayFarer
# #167281791 Setup database

**What does this PR do?**
create a postgres database connection

**Description of the task to be completed?**
Setup an elephant SQL account
Create a postgres DB connection

**How should this be manually tested?**
Clone the repo cd into it and run,
check out the connection string in the .env file
test the endpoints using postman 

**Any background context you want to provide?**
Not applicable

**What are the relevant pivotal tracker stories?**
#167281791
